pkg/controllers/status/cluster_status_controller.go

```diff
@@ -156,7 +156,7 @@ func (c *ClusterStatusController) syncClusterStatus(cluster *clusterv1alpha1.Clu
 	currentClusterStatus := *cluster.Status.DeepCopy()
 
 	// create a ClusterClient for the given member cluster
-	clusterClient, err := c.ClusterClientSetFunc(cluster.Name, c.Client, c.ClusterClientOption)
+	clusterClient, err := c.ClusterClientSetFunc(cluster.Name, c.Client, c.ClusterClientOptions)
```

Comment:
For the usage of [`--cluster-api-qps` and `cluster-api-burst`](https://github.com/karmada-io/karmada/blob/654f146c39f1cf27cb6885358e91e00453638da4/cmd/controller-manager/app/options/options.go#L191-L192):

> Doesn't cover events and node heartbeat apis which rate limiting is controlled by a different set of flags.

I'm not sure the description about `events` and `node heartbeat` is specific to here(cluster status refresh)?

Reply:
@Garrybest Can you help to confirm it? If it is the case, this PR might break the rule.

Reply:
Yeah, I can see [the two flags in kubelet](https://github.com/kubernetes/kubernetes/blob/bcea98234f0fdc5ce4e976758977eec72b13bfd4/cmd/kubelet/app/options/options.go#L501-L502).

Given the `--cluster-api-qps` and `cluster-api-burst` in Karmada are used to specify the limit against `member clusters`, we won't deliver events to member cluster(but we do need to collect health states), so I think we need to update the usage description for disambiguation. How do you say?

By the way, do you think the limit should be applied in case of collecting cluster health states?

Reply:
> So I think we need to update the usage description for disambiguation. How do you say?

Agreed. Actually controllers in karmada does not care about any events and node heartbeat in member cluster.

> By the way, do you think the limit should be applied in case of collecting cluster health states?

Do you mean cluster heartbeat lease? If so, I have the same idea too.